src/scripts.js

```diff
@@ -48,26 +56,67 @@ function displaySleepData() {
   recomendedSleep.innerHTML = `${sleep.calculatePercentageOfRecommendedSleep("2019/09/22")}`
   sleepQuality.innerHTML = `${(((sleep.calculateQualityOfSleepByDay("2019/09/22"))/5)*100).toFixed(2)} %`;
   hoursSlept.innerHTML = `${sleep.calculateHoursSleptByDay("2019/09/22")}`;
-  lastWeekSleep.innerHTML =
```

Comment:
We can for sure figure out a way to make this more dynamic and refactor this function.